.github/workflows/workflow-lint-test-python.yml

```diff
@@ -8,26 +8,47 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
+
+    - name: Checkout code
+      uses: actions/checkout@v3
+
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
         python-version: '3.11.x'
         cache: 'pip'
+
+    - name: Check threshold file
+      id: check_threshold_file
+      uses: JJ/files-present-action@releases/v1
+      with:
+        files: 'coverage_threshold.txt'
```

Comment:
an input parameter to the workflow would be a more direct way to specify the threshold? any particular benefit to having this in a file at the root?

Reply:
yeah I realised it's not really possible without force push which I doubt you would like, I asked thomas and he said he couldn't think of another way either so if you do let me know

Reply:
if it's a parameter, it gets set in the calling workflow that is calling this reusable workflow. This workflow is configured in the repo (such as finesse-backend) owns .github/workflows and can be updated as part of the PR. 

why a force push?

Reply:
how are you going to update the threshold?

Reply:
by editing the calling github-workflow?

Reply:
but how are you going to commit the change? you don't have access to the context in the reusable workflow